cmd/log_integration_test.go

```diff
@@ -16,7 +16,7 @@ func TestLogAndDeletes(t *testing.T) {
 	}
 
 	// Log should have exactly one version
-	err = run.ExecCommand("qri log me/log_test")
+	err = run.ExecCommand("qri log me/log_test --local")
```

Comment:
So you are right to be confused because it shouldn't affect anything aaaaannnnddd it doesn't actually affect anything. I just got a weird false positive when running tests yesterday, it kept hanging or just taking an extraordinarily long time to go, and for whatever reason, when I added the flags the hanging stopped. Removing the flags now.